src/main/scala/com/databricks/spark/redshift/RedshiftWriter.scala

```diff
@@ -98,7 +98,7 @@ private[redshift] class RedshiftWriter(
     val credsString: String = AWSCredentialsUtils.getRedshiftCredentialsString(creds)
     val fixedUrl = Utils.fixS3Url(manifestUrl)
     s"COPY ${params.table.get} FROM '$fixedUrl' CREDENTIALS '$credsString' FORMAT AS " +
-      "AVRO 'auto' DATEFORMAT 'YYYY-MM-DD HH:MI:SS' manifest"
+      "AVRO 'auto' DATEFORMAT 'YYYY-MM-DD HH:MI:SS' manifest ${params.extraCopyOptions}"
```

Comment:
Whoops, just realized that this isn't going to be interpolated properly due to a missing `s` at the start of this string. Will hotfix now. Missed this because the integration tests didn't run for this third-party PR :(